README.md

```diff
@@ -0,0 +1,59 @@
+# Brew formulas for TimescaleDB
+
+The repository that hosts the [brew][1] formulas to install TimescaleDB on
+MacOs.
+
+## Installing from the formula
+
+If you're looking for only installing the TimescaleDB on a MacOs, check how to
+install from our [official guide][2].
+
+## Development
+
+If you want to edit the actual formula locally try:
+
+```bash
+brew edit timescaledb
+```
+
+## Testing
+
+If you want to test a Pull Request in progress in this repository, use the
+following steps:
+
+1. Enter in the formulas' repository folder:
```

Comment:
This is the only numbered step. Maybe just drop the numbering?

Also I'd call it something like "Navigate to the Timescale tap directory"